README.md

```diff
@@ -1,25 +1,38 @@
+# Confidential Containers Documentation
 
-We are interested in integrating existing [Trusted Execution
-Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) (TEE) infrastructure
-support and technologies with the cloud native world. Our focus is to place a kubernetes pod into a
-TEE. 
+Confidential Containers is an open source community working to leverage 
+[Trusted Execution Environments](https://en.wikipedia.org/wiki/Trusted_execution_environment) 
+to protect containers and data and to deliver cloud native 
+confidential computing.
 
-### Find out more...
-- [Documentation](https://github.com/confidential-containers/documentation) : Find out more about
-  the progress to date, overall goals and work in progress.
+**We have a new release every 6 weeks!**
+See [Release Notes](./releases/) or [Quickstart Guide](./quickstart.md)
 
-### Get started quickly... 
-- [Kubernetes Operator for Confidential
-  Computing](https://github.com/confidential-containers/confidential-containers-operator) : An
-  operator to deploy confidential containers runtime (and required configs) on a Kubernetes cluster
+Our key considerations are:
+- Allow cloud native application owners to enforce application security requirements
+- Transparent deployment of unmodified containers
+- Support for multiple TEE and hardware platforms
+- A trust model which separates Cloud Service Providers (CSPs) from guest applications
+- Least privilege principles for the Kubernetes cluster administration capabilities which impact 
+delivering Confidential Computing for guest applications or data inside the TEE
 
-### Contribute...
-- [CONTRIBUTING](CONTRIBUTING.md)
+## Further Detail
+
+[![asciicast](https://asciinema.org/a/eGHhZdQY3uYnDalFAfuB7VYqF.svg)](https://asciinema.org/a/eGHhZdQY3uYnDalFAfuB7VYqF)
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fdocumentation.svg?type=shield)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fdocumentation?ref=badge_shield)
+
+- [Project Overview](./overview.md)
+- [Project Architecture](./architecture.md)
+- [Our Roadmap](./roadmap.md)
+- [Alignment with other Projects](alignment.md)
+
+
+### Associated Repositories
+- [Kubernetes Operator for Confidential Computing](https://github.com/confidential-containers/confidential-containers-operator)
+: An operator to deploy confidential containers runtime (and required configs) on a Kubernetes cluster
 
 
 ## License
-[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fcommunity.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fcommunity?ref=badge_large)
\ No newline at end of file
+[![FOSSA Status](https://app.fossa.com/api/projects/git%2Bgithub.com%2Fconfidential-containers%2Fdocumentation.svg?type=large)](https://app.fossa.com/projects/git%2Bgithub.com%2Fconfidential-containers%2Fdocumentation?ref=badge_large)
```

Comment:
This is the status for the wrong repo. Should point to community.

Reply:
Fixed this